### PR TITLE
Rename cpp check targets to fix warning with ITK caused by global namespace pollution

### PR DESCRIPTION
**Pull request by eile**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,7 +46,7 @@ The following CMake modules can be included in your project:
     * [Coverage](Coverage.cmake) *coverage* target to generate a code coverage
       report as html, if ENABLE_COVERAGE option is also set. Additional compiler
       flags are set in that case, so it should be enabled only for debug builds.
-    * [CppcheckTargets](CppcheckTargets.cmake): *cppcheck* target for
+    * [CommonCPPCheck](CommonCPPCheck.cmake): *cppcheck* target for
       static code analysis. Also adds all cppcheck targets to *tests* target.
     * [clangcheckTargets](clangcheckTargets.cmake): *clangcheck* target for
```
Comment (tribal-tec) [inline README.md]: CommonClangCheck

**Comment by tribal-tec** [comment]
Changelog? Was missing for other commits this week as well I'm afraid (e.g. you +1 a change from @rdumusc)
+1 otherwise

**Comment by eile** [comment]
For this one I figured it's internal, so no ChangeLog needed?

For the other - my bad :-/

**Comment by rdumusc** [comment]
@tribal-tec my bad too; not sure which commit I missed.
@eile I think there are no clear internals in this project and we have seen that some users cherry-pick functions, so it is true that changelog is not a bad idea. (also in this commit it is not clear which was the offending target or function or variable causing global namespace pollution)
Otherwise +1 it is better that way!

**Comment by eile** [comment]
The concrete one was add_cppcheck, ITK has a function of the same name but different impl. Will add changelog.

**Comment by tribal-tec** [comment]
https://github.com/Eyescale/CMake/pull/474 had no changelog

**Pull request merged**